api/src/main/java/io/opentelemetry/trace/Tracer.java

```diff
@@ -141,68 +141,17 @@
   Scope withSpan(Span span);
 
   /**
-   * Returns a {@link Span.Builder} to create and start a new child {@link Span} as a child of to
-   * the current {@code Span} if any, otherwise creates a root {@code Span}.
+   * Returns a {@link Span.Builder} to create and start a new child {@link Span}.
```

Comment:
Not only child. As your comment from the Builder says you can also create root Spans. Maybe it makes sense to just say "a new Span"

Reply:
"a new Span" works. Will update.